examples/reference/panes/VTKVolume.ipynb

```diff
@@ -0,0 +1,101 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import panel as pn\n",
+    "pn.extension('vtk')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The ``VTKVolume`` pane renders 3d volumetric data defined on regular grids. It can be directly a 3d numpy array or a vtkVolume. It allows to interactively define the color opacity to look through the volume.\n",
```

Comment:
```suggestion
    "The ``VTKVolume`` pane renders 3d volumetric data defined on regular grids. It may be constructed from a 3D NumPy array or a `vtkVolume`. The pane provides interactive control over the color opacity to look through the volume.\n",
```